docs/index.html

```diff
@@ -3082,9 +3097,9 @@ <h4 id="_q_i_get_an_error_message_saying_that_a_library_is_not_registered_when_i
 </div>
 </div>
 <div class="sect2">
-<h3 id="_sparx_enterprise_architect">3.3. Sparx Enterprise Architect</h3>
+<h3 id="_sparx_enterprise_architect">4.3. Sparx Enterprise Architect</h3>
 <div class="sect3">
-<h4 id="_q_sparx_enterprise_architect_is_a_windows_tool_isn_t_it">3.3.1. Q: Sparx Enterprise Architect is a windows tool, isn&#8217;t it?</h4>
+<h4 id="_q_sparx_enterprise_architect_is_a_windows_tool_isn_t_it">4.3.1. Q: Sparx Enterprise Architect is a windows tool, isn&#8217;t it?</h4>
```

Comment:
This file needs to be re-generated, as it's missing latest fixes.
I was thinking to make a task to copy manual.html to index.html somehow automatically, otherwise it's easy to forget.

Reply:
I once thought that Travis could do that, but seems to be not easy...